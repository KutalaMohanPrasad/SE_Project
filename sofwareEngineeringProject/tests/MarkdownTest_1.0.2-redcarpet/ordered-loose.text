Loose using tabs:

1.	First

2.	Second

3.	Third

and using spaces:

1. One

2. Two

3. Three
