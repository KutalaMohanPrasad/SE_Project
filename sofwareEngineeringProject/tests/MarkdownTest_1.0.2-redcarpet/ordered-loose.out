<p>Loose using tabs:</p>

<ol>
<li><p>First</p></li>
<li><p>Second</p></li>
<li><p>Third</p></li>
</ol>

<p>and using spaces:</p>

<ol>
<li><p>One</p></li>
<li><p>Two</p></li>
<li><p>Three</p></li>
</ol>